measurements/api/private.py

```diff
@@ -165,169 +161,42 @@ def api_private_countries():
     return jsonify({"countries": country_list})
 
 
-# XXX Everything below here are ghetto hax to support legacy OONI Explorer
-@api_private_blueprint.route("/blockpages", methods=["GET"])
-def api_private_blockpages():
-    probe_cc = request.args.get("probe_cc")
-    if probe_cc is None:
-        raise BadRequest("missing probe_cc")
-
-    # Fastpath for blockpages
-    s = (
-        select([sql.text("SUM(confirmed_count) as confirmed_count")])
-        .where(sql.text("probe_cc = :probe_cc"))
-        .select_from(sql.table("ooexpl_wc_confirmed"))
-    )
-    confirmed_count = current_app.db_session.execute(
-        s, {"probe_cc": probe_cc}
-    ).fetchone()[0]
-    if confirmed_count == 0:
-        return jsonify({"results": []})
-
-    q = (
-        current_app.db_session.query(
-            Report.report_id.label("report_id"),
-            Report.probe_cc.label("probe_cc"),
-            Report.probe_asn.label("probe_asn"),
-            Report.test_start_time.label("test_start_time"),
-            Input.input.label("input"),
-        )
-        .join(Measurement, Measurement.report_no == Report.report_no)
-        .join(Input, Measurement.input_no == Input.input_no)
-        .filter(Measurement.confirmed == True)
-        .filter(Report.test_name == "web_connectivity")
-        .filter(Report.probe_cc == probe_cc)
-        .limit(100)
-    )
+# Deprecated endpoints for legacy OONI Explorer
 
-    results = []
-    for row in q:
-        results.append(
-            {
-                "report_id": row.report_id,
-                "probe_cc": row.probe_cc,
-                "probe_asn": "AS{}".format(row.probe_asn),
-                "test_start_time": row.test_start_time,
-                "input": row.input,
-            }
-        )
 
-    return jsonify({"results": results})
+@api_private_blueprint.route("/blockpages", methods=["GET"])
+def api_private_blockpages():
+    abort(404)
```

Comment:
Maybe we also want to log this in a structured way? Though OTOH perhaps 404 is enough as we can easily parse nginx logs looking for that.

Reply:
We are not logging other calls e.g. /test_names (except from the Nginx side). Also the deprecated calls seemed to have no traffic at all.